app/ts/utils/abi.ts

```diff
@@ -81,3 +81,18 @@ export const CompoundGovernanceAbi: InterfaceAbi = [
 export const CompoundTimeLock: InterfaceAbi = [
 	'function executeTransactions(address[] memory targets, uint[] memory values, string[] memory signatures, bytes[] memory datas, uint eta) public payable'
 ]
+
+function getStringBetweenParentheses(inputString: string): string | undefined {
+	const regex = /\(([^)]+)\)/
```

Comment:
I believe this will do the trick, and also work with `()` (returning empty string).  Note that the `?` tells the regex to not be greedy, which will make it stop as soon as it runs into the closing `)` since that can be matched by the next item.
```suggestion
	const regex = /\((.*?)\)/
```